test/data/ansible/check_ostree.yaml

```diff
@@ -488,6 +488,8 @@
         - name: failed count + 1
           set_fact:
             failed_counter: "{{ failed_counter | int + 1 }}"
+      # NOTE: disable on RHEL 9.0 -- failing for unknown reason -- see rhbz#2053469
+      when: (ansible_facts['distribution'] != 'RedHat' or ansible_facts['distribution_version'] | int != 9.0)
```

Comment:
@achilleas-k do we have a Jira ticket to revert this once the bug gets fixed? If not please open one with the `[QE] ` prefix otherwise we'll forget.